File: JAVA/RemoveDuplicate.java
public class RemoveDuplicate {
    public static void main(String[] args) {
        int[] arr={1,2,1,8,1,2,8};
        boolean[] visited=new boolean[arr.length];

        for(int i=0; i<arr.length; i++)
        {
            if(visited[i])
                continue;
            System.out.println(arr[i]+ " ");
            for(int j=i+1; j<arr.length; j++)
            {
                if(arr[i]==arr[j])
                {
                    visited[j]=true;
                }
            }
        }
    }
}

File: JS/RemoveDuplicate.js
function removeDuplicate(arr)
{
    let unique=[]
    for(let i=0; i<arr.length; i++)
    {
        let isfound=false
        for(let j=0; j<unique.length; j++)
        {
            if(arr[i]==unique[j])
            {
                isfound=true
                break
            }
        }
        if(isfound==false)
            unique.push(arr[i])
    }
    return unique
}
console.log(removeDuplicate([0,0,5,4,8,1,2,1,8]))